samples/meetings-app-icon-badging/csharp/InMeetingNotifications/Manifest/manifest.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.15/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.15",
+  "version": "1.0.0",
+  "id": "{{Microsoft-App-Id}}",
+  "packageName": "com.microsoft.teams.appiconbadging",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://microsoft.com",
+    "privacyUrl": "https://microsoft.com/privacy",
+    "termsOfUseUrl": "https://microsoft.com/termsofuse"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "Meetings-App-Icon-Badging",
+    "full": "Show App Icon Badging In Teams Meetings"
+  },
+  "description": {
+    "short": "App Icon Badging In-Meeting Notification demo",
+    "full": "This sample app shows how to send app icon badging in Teams meetings using targeted notifications API."
```

Comment:
This sample app shows how to send app icon badging in Microsoft Teams meetings using targeted notifications API.

Reply:
fixed